lang/java/avro/src/test/java/org/apache/avro/TestSchemaBuilder.java

```diff
@@ -111,14 +112,14 @@ void objectProps() {
     assertTrue(s.getObjectProp("intProp") instanceof Integer);
     assertTrue(s.getObjectProp("longProp") instanceof Long);
     assertEquals(Long.MAX_VALUE, s.getObjectProp("longProp"));
-    assertTrue(s.getObjectProp("floatProp") instanceof Double);
+    assertTrue(s.getObjectProp("floatProp") instanceof Float);
     // float converts to double
```

Comment:
Remove?

Reply:
? Double is replaced by Float for "floatProp"; i don't understand the point of removing this line ?

Reply:
I intended the comment: it's no longer converted, so the comment is outdated.